internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -5933,6 +5936,237 @@ func TestHandleTSpendVotes(t *testing.T) {
 	}})
 }
 
+func TestHandleGetVoteInfo(t *testing.T) {
+	t.Parallel()
+
+	v0 := uint32(0)
+	v7 := uint32(7)
+
+	okAgendas := []types.Agenda{{
+		ID:             "headercommitments",
+		Description:    "Enable header commitments as defined in DCP0005",
+		Mask:           6,
+		StartTime:      1567641600,
+		ExpireTime:     1599264000,
+		Status:         "started",
+		QuorumProgress: 0.01984126984126984,
+		Choices: []types.Choice{
+			{
+				ID:          "abstain",
+				Description: "abstain voting for change",
+				Bits:        0,
+				IsAbstain:   true,
+				IsNo:        false,
+				Count:       20,
+				Progress:    0.2,
+			}, {
+				ID:          "no",
+				Description: "keep the existing consensus rules",
+				Bits:        2,
+				IsAbstain:   false,
+				IsNo:        true,
+				Count:       10,
+				Progress:    0.1,
+			}, {
+				ID:          "yes",
+				Description: "change to the new consensus rules",
+				Bits:        4,
+				IsAbstain:   false,
+				IsNo:        false,
+				Count:       70,
+				Progress:    0.7,
+			},
+		},
+	}}
+
+	thresholdDefinedAgendas := []types.Agenda{{
+		ID:             "headercommitments",
+		Description:    "Enable header commitments as defined in DCP0005",
+		Mask:           6,
+		StartTime:      1567641600,
+		ExpireTime:     1599264000,
+		Status:         "defined",
+		QuorumProgress: 0,
+		Choices: []types.Choice{
+			{
+				ID:          "abstain",
+				Description: "abstain voting for change",
+				Bits:        0,
+				IsAbstain:   true,
+				IsNo:        false,
+				Count:       0,
+				Progress:    0,
+			}, {
+				ID:          "no",
+				Description: "keep the existing consensus rules",
+				Bits:        2,
+				IsAbstain:   false,
+				IsNo:        true,
+				Count:       0,
+				Progress:    0,
+			}, {
+				ID:          "yes",
+				Description: "change to the new consensus rules",
+				Bits:        4,
+				IsAbstain:   false,
+				IsNo:        false,
+				Count:       0,
+				Progress:    0,
+			},
+		},
+	}}
+
+	okResult := types.GetVoteInfoResult{
+		CurrentHeight: 432100,
+		StartHeight:   431488,
+		EndHeight:     439551,
+		Hash:          "000000000000000023455b4328635d8e014dbeea99c6140aa715836cc7e55981",
+		VoteVersion:   7,
+		Quorum:        4032,
+		TotalVotes:    0,
+		Agendas:       okAgendas,
+	}
+
+	thresholdDefinedResult := types.GetVoteInfoResult{
+		CurrentHeight: 432100,
+		StartHeight:   431488,
+		EndHeight:     439551,
+		Hash:          "000000000000000023455b4328635d8e014dbeea99c6140aa715836cc7e55981",
+		VoteVersion:   7,
+		Quorum:        4032,
+		TotalVotes:    0,
+		Agendas:       thresholdDefinedAgendas,
+	}
+
+	testRPCServerHandler(t, []rpcTest{
+		{
```

Comment:
Formatting is off here - this can be moved to the line above and then have the below shifted left like in the other tests